src/amo/components/AddonReviewList/index.js

```diff
@@ -204,7 +173,11 @@ export class AddonReviewListBase extends React.Component {
         <CardList>
           <ul>
             {allReviews.map((review, index) => {
-              return this.renderReview({ review, key: String(index) });
+              return (
+                <li className="AddonReviewList-li" key={String(index)}>
```

Comment:
I'm not sure we use this class anywhere anymore (based on its removal from the stylesheet below) and it's a bit awkwardly named anyway. I'd be fine just removing it if it isn't used anywhere.